pkg/controller/manifestwork/manager.go

```diff
@@ -130,6 +131,26 @@ func add(importSecretInformer cache.SharedIndexInformer, mgr manager.Manager, r
 		return err
 	}
 
+	if err := c.Watch(
```

Comment:
I don't think we need watch addon. 
only need to delete all addon when the cluster is deleting.

Reply:
Add this to trigger the reconcile to move forward at [here](https://github.com/stolostron/managedcluster-import-controller/pull/161/files#diff-768295ca0279c534580a99162d76fb98bc39e2b35f905dfaa34ca81d815b4574R174), otherwise, we need to wait the next resync period?

Reply:
ok. looks good to me.